src/assets/items/items.ts

```diff
@@ -82,8 +82,15 @@ export const items: { [id: string]: Item } = {
         value: 250,
         description: "You lucky motherfucker",
         type: ItemType.material,  
+    },
+    MinecraftSteveDiamondPickaxe:{
```

Comment:
This is a weapon so please define it in the weapons.ts file instead (btw it is also missing damage despite being a weapon).